Sources/CobSpec/main.swift

```diff
@@ -35,4 +35,6 @@ routesTable.addRoute(route: Route(url: "/method_options", method: HttpMethod.pos
 routesTable.addRoute(route: Route(url: "/method_options2", method: HttpMethod.get, action: nullAction))
 routesTable.addRoute(route: Route(url: "/method_options2", method: HttpMethod.options, action: nullAction))
 
+routesTable.addRoute(route: Route(url:"/logs", method: HttpMethod.get, action: nullAction, realm: "basic-auth", authorization: "YWRtaW46aHVudGVyMg=="))
```

Comment:
It would be nice to move the authorization token into a config file and then populate it in the code. Generally speaking, you do not want to hard-code any tokens / passwords / etc in your code. Technically, you do not want to commit any config files with passwords / tokens / secrets into your version control system either, but it is ok here since this is a toybox app. If you want to be very strict about it, you can use `http://git-secret.io/` to protect your info.